app/controllers/unattended_controller.rb

```diff
@@ -61,7 +61,13 @@ def render_template(type)
     # Compatibility with older URLs
     type = 'iPXE' if type == 'gPXE'
 
-    if (config = @host.provisioning_template({ :kind => type }))
+    if type == 'local_boot'
```

Comment:
Please extract to a separate private method, `find_config_template(type)` or similar